packages/apps/plugins/plugin-space/.storybook/preview-head.html

```diff
@@ -0,0 +1,17 @@
+<script>
+  window.global = window;
+</script>
+<style>
+  html, body {
+    margin: 0 !important;
+    padding: 0 !important;
```

Comment:
I think this is meant to be achieved with the `isFullscreen` flag: https://storybook.js.org/docs/react/configure/features-and-behavior